api/tests/integration/edge_api/identities/test_edge_identity_featurestates_view.py

```diff
@@ -253,6 +290,71 @@ def test_edge_identities_create_featurestate(
     assert args[0] == expected_identity_document
 
 
+def test_edge_identities_with_identifier_create_featurestate(
+    admin_client,
+    environment,
+    environment_api_key,
+    identity_document_without_fs,
+    dynamo_wrapper_mock,
+    feature,
+):
+    # Given
+    dynamo_wrapper_mock.get_item.return_value = identity_document_without_fs
+    identity_identifier = identity_document_without_fs["identifier"]
+    url = reverse(
+        "api-v1:environments:edge-identities-with-identifier-featurestates-list",
+        args=[environment_api_key],
+    )
+    expected_feature_state_value = "random_value"
+    expected_fs_enabled = True
+    data = {
+        "multivariate_feature_state_values": [],
+        "enabled": expected_fs_enabled,
+        "feature": feature,
```

Comment:
I'm guessing this is the numeric feature id, right? Should we allow users to provide the feature name here?